src/main/webapp/script.js

```diff
@@ -19,15 +19,26 @@ function getRecipeInfo() {
   params.append('image', image);
   const request = new Request('/dishAnalysis', {method: "POST", body: params});
   fetch(request).then(response => response.json()).then((recipeListInfoJson) => {
-    //set top options to radio buttons
-    document.getElementById("first-option-label").innerText = recipeListInfoJson[0];
-    document.getElementById("second-option-label").innerText = recipeListInfoJson[1];
-    //set radio button values
-    document.getElementById("first-option").value = recipeListInfoJson[0];
-    document.getElementById("second-option").value = recipeListInfoJson[1];
+    if(recipeListInfoJson["length"] > 1) {
```

Comment:
What exception/error case are you handling here? When vision API returns 0 viable canidates? I think we should handle that case directly

Reply:
When the vision API returns either 0 or 1 viable candidates - however with all of the images I tested they all had at least two, so this is only for the rare exception